core/blockchain.go

```diff
@@ -1427,6 +1427,41 @@ func (bc *BlockChain) reorgNeeded(localBlock *types.Block, localTd *big.Int, ext
 	return reorg
 }
 
+// pruneBlockSidecars prunes the sidecars of blocks that are older than the keep period
+func (bc *BlockChain) pruneBlockSidecars(db ethdb.KeyValueWriter) {
```

Comment:
We should follow the specification in the design document:
"When inserting a new canonical block, prune the blobs of block at: lastest - 518,400 (around 18 days)"

Reply:
Oh I missed that, updating. Btw do you think that we should also change the logic for skipping blob check, which is currently checking using the time of 18 days?